docs-chef-io/content/workstation/getting_started.md

```diff
@@ -318,93 +323,114 @@ See [SSL Certificates](/chef_client_security/#ssl-certificates) for
 more information about how knife and Chef Infra Client use SSL
 certificates generated by the Chef Infra Server.
 
-## Verify Server Communication
+### From Hosted Chef or Chef Manage
```

Comment:
The problem we're going to run into here is that the starter kit steps produce a config.rb and we want users to use the credentials file. This is a :( situation